modules/package.nix

```diff
@@ -177,5 +177,9 @@ in {
       type = nullOr str;
       default = null;
     };
+    patches = mkOption {
+      type = listOf path;
```

Comment:
This should probably be `listOf (either package path)`, in order to support `fetchpatch`.